### ISSUE TITLE
Damaged routes after loading saved game

### ISSUE THREAD

**Issue by holmas** [VictorPhilipp/Cities-Skylines-Traffic-Manager-President-Edition#244]
Every time after loading a saved game several public transport routes are damaged. This is because extra stops have been inserted at strange places. So far I have been able to repair the route by clearing the additional stops. But this time the additional stop is inaccessible. The attachment contains 2 screenshots and the saved game. The first screenshot shows the broken monorail route (the black one) and the second one shows the absolutely strange place of the additional stop.

By the way: thank you soo much for this incredible work

[possible_tmpe_issue.zip](https://github.com/VictorPhilipp/Cities-Skylines-Traffic-Manager-President-Edition/files/2993494/possible_tmpe_issue.zip)

**Comment by holmas** [comment]
This time it's happen after modifying/building any streets. I've connected a new street to an existing one to form a T-crossing. The existing street will be used by a bus route (not the later damaged one). I saw very shortly the "Broken-Route-Icon" and the orange route was damaged afterwards (far away from the building place). Hopefully, you have enough material to find the possible error.

[possible_tmpe_issue_2.zip](https://github.com/VictorPhilipp/Cities-Skylines-Traffic-Manager-President-Edition/files/2993876/possible_tmpe_issue_2.zip)
